### ISSUE TITLE
Pass the initialize blockNumber to the worker so he can wait until he retrieves the chainTaskId

### ISSUE THREAD

**Issue by jeremyjams** [iExecBlockchainComputing/iexec-core#184]
The worker is running on a node which is not quite synchronized, hence fail to retrieve the chainTaskId which is not yet existing in his node.

The scheduler should pass the initialize blockNumber to the worker so he can wait until he sees these block.

```
2019-02-07 15:59:36.205  INFO 1 --- [pool-7-thread-1] c.i.w.replicate.ReplicateDemandService   : The contribution contribAuth is valid [chainTaskId:0xc64d7695f5d251cc94b3643f9b3dccc3b924b291f59f402c9c265e52566b6603]
2019-02-07 15:59:36.206  INFO 1 --- [pool-7-thread-1] c.i.worker.pubsub.SubscriptionService    : Subscribed to topic [chainTaskId:0xc64d7695f5d251cc94b3643f9b3dccc3b924b291f59f402c9c265e52566b6603, topic:/topic/task/0xc64d7695f5d251cc94b3643f9b3dccc3b924b291f59f402c9c265e52566b6603]
2019-02-07 15:59:36.723 ERROR 1 --- [pool-7-thread-1] com.iexec.common.chain.ChainUtils        : Failed to get ChainApp [chainAppId:0x0000000000000000000000000000000000000000]
2019-02-07 15:59:36.723 ERROR 1 --- [pool-7-thread-1] com.iexec.common.chain.ChainUtils        : Failed to get ChainDeal [chainDealId:0x0000000000000000000000000000000000000000000000000000000000000000]
2019-02-07 15:59:36.723  INFO 1 --- [pool-7-thread-1] c.i.w.replicate.ReplicateDemandService   : Failed to retrieve AvailableReplicate, ChainDeal error  [chainTaskId:0xc64d7695f5d251cc94b3643f9b3dccc3b924b291f59f402c9c265e52566b6603]
2019-02-07 15:59:36.723  INFO 1 --- [pool-7-thread-1] c.i.w.replicate.ReplicateDemandService   : Failed to retrieveAvailableReplicateModelFromContribAuth [chainTaskId:0xc64d7695f5d251cc94b3643f9b3dccc3b924b291f59f402c9c265e52566b6603]

```

**Comment by Ugo** [comment]
After thinking a bit about this problem, we should do it the other way around: the worker should come up with its latest available block number and the scheduler should give it only jobs that it already can see on the blockchain. If we do this, we don't need to implement any waiting mechanism for the worker (then no need to make the worker smarter that it is supposed to be!).

**Comment by Ugo** [comment]
Fixed in the PR  #196
